cli/src/main/java/com/devonfw/tools/ide/property/Property.java

```diff
@@ -404,19 +416,20 @@ protected boolean applyValue(String argValue, boolean lookahead, CliArguments ar
    * @param commandlet the {@link Commandlet} owning this {@link Property}.
    * @param collector the {@link CompletionCandidateCollector}.
    */
-  protected void complete(CliArgument argument, CliArguments args, IdeContext context, Commandlet commandlet, CompletionCandidateCollector collector) {
+  protected void complete(String nameNormalized, CliArgument argument, CliArguments args, IdeContext context, Commandlet commandlet,
+      CompletionCandidateCollector collector) {
```

Comment:
Please adjust Javadoc, normalizedName is missing.

Reply:
Done. Also aligned the parameter name (`nameNormalized` vs. `normalizedName`). Also found another undocumented method from someone else and added JavaDoc.